DetDescr/src/DetDescr/EcalGeometry.cxx

```diff
@@ -55,12 +55,12 @@ EcalGeometry::EcalGeometry(const framework::config::Parameters& ps)
   moduler_ = ps.getParameter<double>("moduleMinR");
   gap_ = ps.getParameter<double>("gap");
   nCellRHeight_ = ps.getParameter<double>("nCellRHeight");
-  verbose_ = ps.getParameter<int>("verbose");
   cornersSideUp_ = ps.getParameter<bool>("cornersSideUp");
   layer_shift_x_ = ps.getParameter<double>("layer_shift_x");
   layer_shift_y_ = ps.getParameter<double>("layer_shift_y");
   layer_shift_odd_ = ps.getParameter<bool>("layer_shift_odd");
   layer_shift_odd_bilayer_ = ps.getParameter<bool>("layer_shift_odd_bilayer");
+  si_thickness_ = ps.getParameter<double>("si_thickness");
```

Comment:
These parameters are pulled from `DetDescr/python/EcalGeometry.py` python config class, so this change will need to be propagated there.

Reply:
Oh yes, I actually did this, just didnt push, my bad!